### DISCUSSION TITLE
Total cost per day

### DISCUSSION THREAD

**Discussion by Damazinho** [JaccoR/hass-entso-e#46] [Q&A]
Has somebody created something to have a sensor that counts the total cost of the day? Since I have a dynamic energy price i can't figure out how to calculate this. I want to use this for the push message I always received from HA through a automation when I had static energy prices at the end of the day.

**Comment by rest77** [comment]
Check out the energy dashboard. You can connect the actual price to your import of electricity. Then HA keeps track of the cost.

**Reply by Damazinho** [reply]
I know. So how do I put the value in a automation for the push message to my phone?
